source/includes/api/_vocabulary.md

```diff
@@ -1,694 +1,1297 @@
-```
+Assuming the operation was successful, the response will contain a list of all vocabularies and their respective tags
+associated with the specified resource, for all applications - including the vocabulary that was deleted.
 
-> Getting a single relationship between a Vocabulary and a Widget
+If you want to delete vocabularies/tags for your resources, you must have the necessary user account permissions.
+Specifically:
 
-```shell
-curl -X GET https://api.resourcewatch.org/v1/dataset/<dataset-id>/widget/<widget-id>/vocabulary/<vocabulary-id>
-```
-
-> Getting a single relationship between a Vocabulary and a Layer
+- the user must be logged in and belong to the same application as the resource
+- the user must match one of the following:
+    - have role `ADMIN`
+    - have role `MANAGER` and be the resource's owner (through the `userId` field of the resource)
 
-```shell
-curl -X GET https://api.resourcewatch.org/v1/dataset/<dataset-id>/layer/<layer-id>/vocabulary/<vocabulary-id>
-```
+When deleting vocabulary/tags for a resource, the dataset id specified in the URL is validated, and the requests fail if
+a dataset with the given id does not exist. When deleting a vocabulary for a widget or layer, you should ensure you
+specify the dataset id that matches that of the widget/layer, as that validation is not done automatically - this is a
+known limitation of the current implementation and may be modified at any time.
 
-## Getting resources (COMMON USE CASE)
+#### Errors for deleting a single vocabulary/tags for a resource
 
-There is also an endpoint available which accepts requests just indicating resource type and
-the desired vocabulary-tag matching.
+Error code     | Error message  | Description
+-------------- | -------------- | --------------
+401            | Unauthorized   | You are not authenticated.
+403            | Forbidden      | You are trying to delete a vocabulary for a resource whose `application` value is not associated with your user account.
+404            | 404 - {\"errors\":[{\"status\":404,\"detail\":\"Dataset with id `<dataset id>` doesn't exist\"}]}      | You are trying to create a vocabulary for a dataset that doesn't exist.
+404            | Relationship between `<vocabulary id> and dataset - `<dataset id> and dataset: `<dataset id> doesn't exist      | You are trying to delete a vocabulary that doesn't exist.
 
-It currently supports DOUBLE-OR pattern matching, meaning the API will return
-all resources that at least have one (or more) tags in a particular vocabulary. At the same time it will also apply an OR to the entities of other vocabulary-tag matchings.
+### Deleting all vocabulary/tags for a resource
 
-The query has to be set in the url by the query params.
+> Deleting all vocabularies/tag for a dataset
 
 ```shell
-curl -X GET https://api.resourcewatch.org/v1/dataset/vocabulary/find
+curl -X DELETE https://api.resourcewatch.org/v1/dataset/<dataset-id>/vocabulary/<vocabulary-id>
```

Comment:
Confused...this is identical to the single deletion case

Reply:
Along with the other examples in this section

Reply:
copy-pasting gone wrong. will fix it